docs/tutorials/services/plan-motion-with-arm-gripper.md

```diff
@@ -80,10 +80,35 @@ if err != nil {
 {{% /tab %}}
 {{< /tabs >}}
 
-Once the Motion Service can be accessed, some familiar features become available.
-The Motion Service has a method that can get the *pose* of a component relative to a [*reference frame*](../../../services/frame-system/).
-In the tutorial where we interacted with an arm component, we used the `EndPosition` method to determine the pose of the end effector of `myArm`.
-The `GetPose` method provided by the Motion Service serves a similar function to `EndPosition`, but allows for querying of pose data with respect to other elements of the robot (such as another component or the robot's fixed "world" frame).
+The Motion service has a method that can get the *pose* of a component relative to a [*reference frame*](../../../services/frame-system/).
+In the tutorial where we interacted with an arm component, we used the `GetEndPosition` method to determine the pose of the end effector of `myArm`.
+The `GetPose` method provided by the Motion Service serves a similar function to `GetEndPosition`, but allows for querying of pose data with respect to other elements of the robot (such as another component or the robot's fixed "world" frame).
+
+### Get the `ResourceName`
+
+When you use the [arm API](../../../components/arm/#api), you call methods on your arm component itself.
```

Comment:
```suggestion
When you use the [arm API](../../../components/arm/#api), you control your arm {{< glossary_tooltip term_id="resource" text="resource" >}}.
```
Sentence is a little bit awk--suggesting as I think this would help more to introduce the following sentence re. ResourceName!

Reply:
```suggestion
When you use the [arm API](../../../components/arm/#api), you refer to your arm by the `name` you gave it in your component configuration.
```